command/repo.go

```diff
@@ -300,65 +300,83 @@ func repoCreate(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	repo, err := api.RepoCreate(client, input)
-	if err != nil {
-		return err
+	confirmRepoCreate := false
+	promptString := ""
+	if input.OwnerID == "" {
+		promptString = "This will create " + name + " in your personal account. Continue? "
```

Comment:
I personally didn't like this fix but I couldn't figure out a way to get the `${userName}/${repoName}` format. I'd be glad if someone has a way out here.